src/main/java/de/danielbechler/diff/differ/PrimitiveArrayDifferFactory.java

```diff
@@ -0,0 +1,11 @@
+package de.danielbechler.diff.differ;
+
+import de.danielbechler.diff.NodeQueryService;
+
+public class PrimitiveArrayDifferFactory implements DifferFactory {

```

Comment:
No need for the factory when you just add the Differ in the `ObjectDifferBuilder` directly.